plugins/autowarmth.koplugin/main.lua

```diff
@@ -29,6 +29,10 @@ local T = FFIUtil.template
 local Screen = require("device").screen
 local util = require("util")
 
+local math_abs = math.abs
+local math_floor = math.floor
+local math_min = math.min
```

Comment:
I know @NiLuJe used that kind of stuff in blitbuffer.lua which are super hot codepaths.
I'm not sure this is required in this module - and it would create a precedent and one might think this is a good idea and the thing to do, and use that everywhere.

Reply:
Jepp for a singular math.floor i a widget init that alias makes no sense (but does no harm ). 
But here at least we use math.abs, math.floor, ...  quite often on wakeup (from standby ). So it might save some milli Joule of energy?

Reply:
Maybe a comment in the code might dispert your concerns.

Reply:
The consensus is that it might actually be actively harmful on LuaJIT, so, yeah, probably not worth the hassle.